examples/examples.md

```diff
@@ -1747,7 +1661,7 @@ states:
         parameters:
           patientid: "{{ $.patientId }}"
   end:
-    kind: terminate
+    terminate: true
```

Comment:
What does this terminate attribute mean? Just `end: true` doesn't mean the same thing?

Reply:
yeah its different and i will do another pr to describe this better.
"end": true (without the terminate) stops instance execution, but does **not** terminate the instance itself. The instance is still active, meaning that correlated events can still be associated with that instance

"terminate": true terminate the instance completely, meaning its execution completely stops. new comsumed events at that point even if correlated will create a new workflow instance.

hope this helps.